### PR TITLE
Implement wall->desktop events in DesktopStreamer

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by rdumusc** [comment]
I don't mind, but is it really cleaner? If you think so, go for it and also please move up the source of SimpleStreamer main.cpp as well :-)

**Comment by eile** [comment]
done

**Comment by eile** [comment]
It's cleaner because it removes an unnecessary directory, imo.

**Comment by rdumusc** [comment]
+1

File: apps/DesktopStreamer/CMakeLists.txt [hunk 1]
```diff
@@ -28,8 +28,9 @@ if(APPLE)
   set(DESKTOPSTREAMER_ICON_FILE desktopstreamer.icns)
 
   # Required to control AppNap on OSX >= 10.9
```
Comment (rdumusc) [inline apps/DesktopStreamer/CMakeLists.txt]: Update comment, not only for AppNap anymore

**Comment by eile** [comment]
Hold merge until it has been tested on the wall.

**Comment by rdumusc** [comment]
+1 - the only point of detail is to maybe move the injection of the mouse events to a separate class?

**Comment by eile** [comment]
Yes, we should do this once it gets more complex.

**Comment by eile** [comment]
Retest this please

**Comment by rdumusc** [comment]
+1, merge when you're ready

**Pull request merged**